css/style.css

```diff
@@ -151,4 +163,47 @@
     margin-left: 10px;
 }
 
+.modal-bg {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    overflow: auto;
+    height: 100%;
+    width: 100%;
+    background-color: rgba(170, 170, 170, 0);
+    visibility: hidden;
+}
+
+.modal {
+    position: absolute;
+    width: 850px;
+    height: 830px;
+    top: 8%;
+    left: 28%;
+    background-color: white;
+    z-index: 20;
+    margin: 0 auto;
+    z-index: 2;
+}
+.modal--close {
```

Comment:
I think you should rethink naming `modal--close` as it refers in your code to a button, not modal itself. If you meant that it is a block being `modal close`, it should have only one hyphen (but actually it should be treated as an element of a modal, not independent block). Two hyphens means that it is a modifier for a modal